README.md

```diff
@@ -1,6 +1,20 @@
 
 NOTE: This software is only if you want to *provide* the service.
 
+Additional documentation:
+
+- [cloning script][clone-script]
+- [spamgourmet home][spam-home] ([archived][spam-home-arch])
+    - [spamgourmet bulletin board][spam-bbs] ([archived][spam-bbs-arch])
+        - [Developers forum][spam-bbs-dev] ([archived][spam-bbs-dev-arch])
```

Comment:
Just checking, is the indent expected?

![image](https://user-images.githubusercontent.com/2684652/63654434-830b3c80-c77a-11e9-9666-4e41c932fafe.png)

Reply:
Yes, that's intended. It was meant to indicate nesting, i.e., Developers forum within spamgourmet BBS within spamgourmet domain. Is it unclear? In that case, I don't mind making it a flat list.

Reply:
Ah cool, makes sense now. I think the GitHub generated bullet points make it look odd.